locust/util/date.py

```diff
@@ -0,0 +1,5 @@
+from datetime import datetime
```

Comment:
I think this fits better in html.py directly. it isnt super likely that it will be needed anywhere else anyway.

Reply:
I was thinking it might be useful for #2678, having a general utility to manage creating the timestamps would ensure we are consistent everywhere

Reply:
Ok if you foresee the need for it somewhere else too then it is fine!